selfdrive/ui/qt/offroad/settings.h

```diff
@@ -45,14 +57,15 @@ class DevicePanel : public ListWidget {
 signals:
   void reviewTrainingGuide();
 
-private slots:
+protected slots:
   void poweroff();
   void reboot();
   void updateCalibDescription();
 
-private:
+protected:
   Params params;
   ButtonControl *pair_device;
+  QHBoxLayout *power_layout;
```

Comment:
(same as settings.cc) I saw we also did this on our prior split, but I am unsure why do we need it (lack of the context) so i wonder if we really needed it ?